src/main/java/uk/gov/hmcts/darts/common/entity/base/ModifiedBaseEntity.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.common.entity.base;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.UpdateTimestamp;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+
+import java.time.OffsetDateTime;
+
+@MappedSuperclass
+@Getter
+@Setter
+public class ModifiedBaseEntity {
+
+    @UpdateTimestamp
+    @Column(name = "last_modified_ts")
+    private OffsetDateTime lastModifiedDateTime;
```

Comment:
see other comment on _ts naming consistency

Reply:
renamed to timestamp